photon-ml/src/integTest/scala/com/linkedin/photon/ml/util/IOUtilsIntegTest.scala

```diff
@@ -48,7 +48,7 @@ class IOUtilsIntegTest extends SparkTestUtils with TestTemplateWithTmpDir {
       Array(baseDir.toString, DateRange.fromDates("20160101-20160401"), Seq(path1, path2, path3)),
       Array(baseDir.toString, DateRange.fromDates("20160101-20160301"), Seq(path1, path2, path3)),
       Array(baseDir.toString, DateRange.fromDates("20160101-20160201"), Seq(path1, path2)),
-      Array(baseDir.toString, DateRange.fromDates("20160101-20160101"), Seq(path1)),
+      Array(baseDir.toString, DateRange.fromDates("20160101-20160102"), Seq(path1)),
```

Comment:
hmm, we still have some use cases out there that require the start date to be the same as the end date (e.g., in some weekly/monthly data used by feed and jymbii). It looks like the problem is caused by the ```require``` statement in ```DateRange```.

But I will open an issue and address this problem in a separate PR.